bundles/org.openhab.binding.sbus/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -98,6 +98,37 @@
 		</config-description>
 	</thing-type>
 
+	<!-- Contact Device -->
+	<thing-type id="contact">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="udp"/>
+		</supported-bridge-type-refs>
+		<label>Sbus Contact Sensor</label>
```

Comment:
Probably slipped through on the initial PR for the other Thing declarations, but Sbus (binding name) should not be part of this label.
```suggestion
		<label>Contact Sensor</label>
```